docs/webhooks.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Webhooks
+excerpt: Receive HTTP based payloads anytime an event happens on your Fiberplane workspace
+category: 644f7777d0901a002b72fa8a
+slug: webhooks
```

Comment:
```suggestion
slug: creating-and-configuring-webhooks
```

Reply:
id like to keep the main page at `/webhooks` as its referenced that way in both api and fp